examples/bidder/mock_exchange/main.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python2
```

Comment:
Python3 GIT handling is much better than that of Python2 (no console polling in non-interactive scripts).